trade-finance-manager-ui/server/constants/deals.js

```diff
@@ -11,4 +11,5 @@ const TFM_SORT_BY_DEFAULT = {
 module.exports = {
   TFM_SORT_BY,
   TFM_SORT_BY_DEFAULT,
+  PAGE_SIZE: 20,
```

Comment:
Sorry - sticking with convention:
```suggestion
  TFM_PAGE_SIZE: 20,
```